nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -591,36 +629,57 @@ in {
 
   config = mkIf cfg.enable {
     users.extraUsers = [
-      { name = "matrix-synapse";
-        group = "matrix-synapse";
-        home = "/var/lib/matrix-synapse";
+      { name = user;
+        group = group;
+        home = "${cfg.state_path}";
         createHome = true;
         shell = "${pkgs.bash}/bin/bash";
-        uid = config.ids.uids.matrix-synapse;
+        uid = config.ids.uids."${user}";
```

Comment:
isn't `config.ids.uids` a static table? I don't think this is useful, and it brings into question the whole dynamic user name option.

Reply:
Yeah, I suppose it is.  I didn't actually expose the user name, I was just trying to avoid duplication.  i'll put it back the way it was.